tests/Feature/Api/Review/ReviewPhotoApiTest.php

```diff
@@ -56,7 +56,7 @@ public function test_delete_review_photo()
         $review = factory(Review::class)->create();
 
 
-        $response = $this->json('POST', '/api/v1/review-photos', [
+        $response = $this->json('POST', '/api/v1/reviews/photos', [
```

Comment:
No need to make api call. $this->json('POST', '/api/v1/reviews/photos')...
You can use `Storage::put('file.jpg', $contents);` create ReviewPhoto model directly first. After you can call delete photo api and assert data.